### ISSUE TITLE
[XamarinProfiler-Windows]: Accessibility: Color Contrast: MAS1.4.3: "Text" under Count,Size column of All Snapshots table in "Snapshots" tab is not having color contrast ratio of 4.5:1

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#431]
### Pre-Requisite:
1. Install Xamarin Profiler.
2. Launch Profiler through ".mlpd" file.

### Steps to Reproduce

1. Follow the Pre-requisite to launch the xamarin profiler.
2. Select the available instruments to "Allocations" and hit "Start Profiling"button.
3. This will invoke "Allocations" profiling page. Navigate to "Snapshots" tab and invoke it.
4. Verify the color contrast ratio of "Text Present" under Count,Size columns

### Expected Behavior
Color contrast ratio should be minimum 4.5:1
### Actual Behavior
Color contrast ratio is less than 4.5:1 i.e, 1.1:1 - 2.7:1
See the screenshots attached
### User Impact
Low vision user will feel difficult in seeing the text
### Screenshots
![6 a](https://user-images.githubusercontent.com/22956130/41711598-18b0a17a-7566-11e8-9b3c-f03b831e8494.png)
![6 b](https://user-images.githubusercontent.com/22956130/41711603-1ea6661e-7566-11e8-926e-78f2dcfaf61e.png)

### Environment
Microsoft Windows version: 1803(OS Build 17134.112)
Xamarin Profiler version: 1.6.3.342

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS1.4.3
Xamarin-Profiler-Windows
Wipro
